scripts/migrate_logs_branded_preprints.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import sys
+
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.models import NodeLog, PreprintService
+from scripts import utils as script_utils
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+
+logger = logging.getLogger(__name__)
+
+
+def migrate(dry_run=True):
+    node_logs = list(NodeLog.find(
+        Q('action', 'in', [NodeLog.PREPRINT_FILE_UPDATED, NodeLog.PREPRINT_INITIATED]) &
+        Q('params.preprint', 'exists', False)
+    ))
+
+    logger.info('Preparing to migrate {} NodeLogs'.format(len(node_logs)))
+
+    count = 0
+
+    for log in node_logs:
+        preprint = None
+        node_id = log.params.get('node')
+
+        try:
+            preprint = PreprintService.find_one(Q('node', 'eq', node_id))
```

Comment:
As long as this goes out before the branded migration, this shouldn't raise an exception. Otherwise it will definitely raise `MultipleResultsFound`. No action required, just as a note.